### PR TITLE
Hotfix/method regex

### LINKED ISSUE #215: self.fromRichText is not generated into documentation
The `self.fromRichText` method is not being caught by the generate script to create the apostorphe-areas api documentation reference: https://docs.apostrophecms.org/apostrophe/reference/modules/apostrophe-areas. The methods next to it are, and it's not clear what's different with `fromRichText`.

### PR DESCRIPTION
**Pull request by abea**
Resolves #215 

The money is in `945099b`, ("👉 The Fix: captures methods with spaces before the parens"). I generated the references to test the fix and figured it'd be good to get those updates in as well. Reviewing the commits individually will be helpful on this one.

### CODE REVIEW

**Comment by abea** [comment]
@stephenkostas will my generating the references with the script break all the links? Were you fixing those manually?

**Comment by stephenkostas** [comment]
The links should all be fine. I'm not manually updating anything at this point.

File: modules/apostrophe-widgets/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 The base class for all modules that implement a widget, such as
-[apostrophe-rich-text-widgets](../apostrophe-rich-text-widgets/README.md),
-[apostrophe-pieces-widgets](../apostrophe-pieces-widgets/README.md) and
-[apostrophe-video-widgets](../apostrophe-video-widgets/README.md).
+[apostrophe-rich-text-widgets](/modules/apostrophe-rich-text-widgets/),
+[apostrophe-pieces-widgets](/modules/apostrophe-pieces-widgets/) and
+[apostrophe-video-widgets](/modules/apostrophe-video-widgets/).
 
-All widgets have a [schema](/tutorials/schema-guide/schema-guide.md).
+All widgets have a [schema](../../tutorials/getting-started/schema-guide.html).
```
Comment (stephenkostas) [inline modules/apostrophe-widgets/README.md]: It looks like this link got reverted somehow.

Comment (stephenkostas) [reply modules/apostrophe-widgets/README.md]: It should be the /tutorials/schema-guide/schema-guide.md not the .html url

File: modules/apostrophe-widgets/README.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ the appropriate div, the `data` for the widget, and the `options` that
 were passed to the widget.
 
 For example, here is the `public/js/always.js` file for the
-[apostrophe-video-widgets](../apostrophe-video-widgets/README.md) module:
+[apostrophe-video-widgets](../apostrophe-video-widgets/index.html) module:
```
Comment (stephenkostas) [inline modules/apostrophe-widgets/README.md]: This one should also be the README.md, not index.html

**Comment by abea** [comment]
Cool, I'll fix those manually here and in Apostrophe code as well. They're coming from that code base.

File: modules/apostrophe-widgets/README.md (hunk 1 above)
Comment (stephenkostas) [reply modules/apostrophe-widgets/README.md]: For line 7, I'm still seeing the link as "[schema](../../tutorials/getting-started/schema-guide.html)."
When it should be "[schema](/tutorials/schema-guide/schema-guide.md)"

File: modules/apostrophe-widgets/README.md (hunk 2 above)
Comment (stephenkostas) [reply modules/apostrophe-widgets/README.md]: It looks like this one also wasn't fixed at line 106.

**Comment by abea** [comment]
Bumping this.

**Comment by abea** [comment]
Bumping this for @boutell

File: _api-reference-generator/lib/modules/documentation/index.js [hunk 3]
```diff
@@ -413,7 +413,7 @@ module.exports = {
         }
 
         // Make sure it's not commented out
-        var methodRegex = /\n +self\.(\w+)\s*=\s*function\((.*?)\)/g;
+        var methodRegex = /\n +self\.(\w+)\s*=\s*function\s*\((.*?)\)/g;
```
Comment (boutell) [inline _api-reference-generator/lib/modules/documentation/index.js]: 👍

**Pull request merged**